Lesson 15

OnlyReFLeX opened the pull request.

OnlyReFLeX committed:
add admins

OnlyReFLeX committed:
API for questions & answers

OnlyReFLeX committed:
refactoring

vkurennov commented on app/controllers/api/v1/questions_controller.rb:
```diff
@@ -0,0 +1,27 @@
+class Api::V1::QuestionsController < Api::V1::BaseController
+  authorize_resource only: :create
```
Почему авторизация вдруг только на create?

vkurennov commented on app/controllers/api/v1/answers_controller.rb:
```diff
@@ -0,0 +1,32 @@
+class Api::V1::AnswersController < Api::V1::BaseController
```
Где авторизация для контроллера?

vkurennov commented on app/controllers/api/v1/profiles_controller.rb:
```diff
@@ -1,19 +1,9 @@
-
-  respond_to :json
-
+class Api::V1::ProfilesController < Api::V1::BaseController
```
Где авторизация для контроллера?

OnlyReFLeX committed:
fix authorize

OnlyReFLeX committed:
refactoring

Since alias, like def, is a keyword, prefer bareword arguments over symbols or strings. In other words, do alias foo bar, not alias :foo :bar.

vkurennov commented on app/controllers/api/v1/questions_controller.rb:
```diff
@@ -0,0 +1,27 @@
+class Api::V1::QuestionsController < Api::V1::BaseController
+  authorize_resource
```
Авторизацию лучше вызывать после загрузки всех данных

OnlyReFLeX committed:
refactoring
